android/ta_player_plugin/src/main/java/com/applicaster/plugin/televisionacademyplayer/PlayerContract.kt

```diff
@@ -135,7 +139,12 @@ class PlayerContract : BasePlayer(), ApplicationLoaderHookUpI {
     }
 
     override fun executeOnApplicationReady(context: Context?, listener: HookListener?) {
-        BitmovinCastManager.initialize()
+        Log.d("TAG", "executeOnApplicationReady")
+        BitmovinCastManager.getInstance().addMessageReceivedCallback { var1, var2, var3 ->
+            Log.d("TAG", "addMessageReceivedCallback var1 : $var1 var2 : $var2 var3 % $var3 ")
+        }
+        BitmovinCastManager.initialize("F02DA75A", "some_meessage_namespace", ExpandedControlsActivity::class.java)
+        BitmovinCastManager.getInstance().updateContext(context)
```

Comment:
make application ID configurable via zapp please.

Reply:
It was just for debugging purposes, no longer needed. Thanks.